### DISCUSSION TITLE
Unable to load itraconazole model v 1.3. using the beta version

### DISCUSSION THREAD

**Discussion by AyaSaleh90** [Open-Systems-Pharmacology/Forum#996] [Questions & Problems] [answered]
Dear OSP community,

When I try to load the itraconazole model snapshot using the beta version of PK-Sim, I get this error message, will it work only with the 10 version and not with the beta? or is there something I'm not following here.

![image](https://user-images.githubusercontent.com/67606842/138102381-02c76e6a-7496-4c92-bbc4-c84ff34ef394.png)

I appreciate the help you can provide.
Best regards,
Aya

**Comment by Yuri05** [comment, marked as answer]
The beta-version you use (10.0.198) is pretty old and outdated. 
Please use the [newest one (10.0.256)](https://ci.appveyor.com/api/buildjobs/n5isd0w8jln0rk4q/artifacts/pk-sim-portable-setup.zip)

**Reply by AyaSaleh90** [reply]
Thanks a lot for your feedback, and now the model was loaded from the latest snapshot.
